catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -91,25 +103,39 @@ define([
             }
 
             if(!this.get('multiValued')) {
-                if(this.get('type') === 'number'){
+                if(type === 'number'){
```

Comment:
Good suggestions. I have created ticket https://codice.atlassian.net/browse/DDF-2234 to look into this and other refactoring options.